jobs/pulse3d/pulse3d-worker/src/main.py

```diff
@@ -37,38 +39,50 @@ async def process(con, item):
         key = f"{prefix}/{filename}"
     except Exception as e:
         logger.exception("Fetching upload details failed")
-        return ('error', {})
-
+        return ("error", {})
 
     with tempfile.TemporaryDirectory(dir="/tmp") as tmpdir:
         logger.info(f"Downloading {PULSE3D_UPLOADS_BUCKET}/{key} to {tmpdir}/{filename}")
         try:
             s3_client.download_file(PULSE3D_UPLOADS_BUCKET, key, f"{tmpdir}/{filename}")
         except Exception as e:
             logger.exception(f"Failed to download: {e}")
-            return ('error', {})
+            return ("error", {})
 
         try:
             logger.info(f"Starting pulse3d analysis")
             outfile = f"{'.'.join(filename.split('.')[:-1])}.xlsx"
-            prs = PlateRecording.from_directory(f"/{tmpdir}")
-            for r in prs:
-                write_xlsx(r, name=outfile)
+            prs = PlateRecording.from_directory(tmpdir)
```

Comment:
I think for this we might want to do the for loop even though the mantarray will only have 1 recording per zip, when we build out the mybinder replacement users might upload a zip file with multiple recordings in it